memgpt/agent.py

```diff
@@ -431,7 +431,7 @@ def verify_first_message_correctness(self, response, require_send_message=True,
         function_name = function_call.get("name") if function_call is not None else ""
         if require_send_message and function_name != "send_message" and function_name != "archival_memory_search":
             printd(f"First message function call wasn't send_message or archival_memory_search: {response_message}")
-            return False
+            return True

```

Comment:
@Maximilian-Winter is there a reason for this diff?

Reply:
Sorry, I added that, because the LLM always wanted to search its memory in a endless loop.